tools/release/adot-operator-images-mirror/get-operator-tag.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+COLLECTOR_TAG=$(cat go.mod | grep "go.opentelemetry.io/collector " | cut -d " " -f2)
+COLLECTOR_TAG_WITHOUT_PATCH_VER=$(echo $COLLECTOR_TAG | sed 's/\..$//')
```

Comment:
```suggestion
COLLECTOR_TAG=$(cat go.mod | grep "go.opentelemetry.io/collector " | cut -d " " -f2 | sed 's/\..$//)
```

I don't think the full tag is used anywhere anymore, so might as well collapse this.  

Also the regex needs to catch all characters after the final `.`.  Right now it will break if we ever get a `vX.Y.10`.  If it were PCRE then `s/\.[^.]+$//` would work, but not sure `sed` supports that negative range format.  If you can make it work quickly, great, otherwise leave a comment and maybe create an issue to tack the tech debt.  I doubt we're going to get ten patch releases on a single version of the collector any time soon.

Reply:
Collapsed, and changed the `sed` to another `cut` (`cut -d "." -f 1,2`) so that it instead delimits by `.` and gets the first two fields (major.minor) but not the third (patch)

Reply:
now you're thinking with pipelines! :)